amethyst_assets/src/progress.rs

```diff
@@ -188,13 +200,13 @@ impl Tracker for () {
         error: Error,
     ) {
         error!(
-            "error loading handle {}, {}, with name {}, caused by: {:?}",
+            "Error loading handle {}, {}, with name {}, caused by: {:?}",
             handle_id, asset_type_name, asset_name, error
         );
         error
             .iter()
             .skip(1)
             .for_each(|e| error!("caused by: {:?}", e));
```

Comment:
Do you want to do the same thing as you did above to avoid the out of order errors?

Reply:
yes